setup.py

```diff
@@ -28,10 +28,10 @@ def readme():
     # NCATS "homepage"
     url='https://www.dhs.gov/cyber-incident-response',
```

Comment:
It's not great, but this page is probably the most representative page we have going for us right now:

https://www.us-cert.gov/resources/ncats

Reply:
Fixed in e2c34d3.